Tests/UseCaseTest.php

```diff
@@ -86,41 +86,6 @@ protected function getRouterMock()
         return $router;
     }
 
-    protected function getMockRequest($theme, $cookieReturnValue = 'cookie', $userAgent = 'autodetect')
```

Comment:
the old tests needlessly mocked the Request. since the Request is just a value object, its easier to just set the state as necessary. see below.